packages/docs/src/en/directives/transition.md

```diff
@@ -46,15 +46,17 @@ You can override these defaults with modifiers attached to `x-transition`. Let's
 <a name="customizing-duration"></a>
 ### Customizing duration
 
+Initially, the duration is set to be 150 milliseconds when entering, and 75 milliseconds when leaving.
+
 You can configure the duration you want for a transition with the `.duration` modifier:
 
 ```alpine
 <div ... x-transition.duration.500ms>
 ```
 
-The above `<div>` will transition for 500 milliseconds when entering, and 250 milliseconds when leaving.
+The above `<div>` will transition for 500 milliseconds when entering, and 500 milliseconds when leaving.
```

Comment:
Sorry, am I wrong in think that this shouldn't be changed to 500? Please clarify - thanks!

Reply:
I found out that as soon as you specify a duration, it takes over for both entering and leaving.

You can see what I'm talking about in [this CodePen](https://codepen.io/agenordebriat/pen/vYpXEjo).

In this example, I bumped the duration to `5000ms` instead of the `500ms` on the doc and you can clearly see it obviously takes `5000ms` to appear, but also `5000ms` to disappear and not `2500ms` as currently stated in the doc.

The halved duration only exists for the default value, when you don't specify any duration.

Reply:
Ok great, thanks!